src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.courses.domain;
+
+public class Image {
+    private String fileName;
+    private String contentType;
+    private long sizeInBytes;
+    private int width;
+    private int height;
+    private static final int IMAGE_SIZE_1MB = 1024 * 1024;
+    private static final int WIDTH_LIMIT = 300;
+    private static final int HEIGHT_LIMIT = 200;
+    private static final double REQUIRED_ASPECT_RATIO = 1.5;
+    private static final double ASPECT_RATIO_TOLERANCE = 0.01;
```

Comment:
```suggestion
    private static final int IMAGE_SIZE_1MB = 1024 * 1024;
    private static final int WIDTH_LIMIT = 300;
    private static final int HEIGHT_LIMIT = 200;
    private static final double REQUIRED_ASPECT_RATIO = 1.5;
    private static final double ASPECT_RATIO_TOLERANCE = 0.01;
    
    private String fileName;
    private String contentType;
    private long sizeInBytes;
    private int width;
    private int height;
```

관례에 따라 상수는 멤버변수보다 먼저 배치하는데 
아래에 배치하는 스타일이신가요?